### PR TITLE
Added CoWin OTP retriever android app as an alternative to IFTTT

### PR DESCRIPTION
**Pull request by ujaiswal**
People familiar with APK installation through unknown sources can use it as an alternative to IFTTT.
I have created an android app which implements the sms trigger in the source code itself with a simple UI.

Try this apk: https://github.com/ujaiswal/covid-vaccine-booking/blob/main/CoWinOtpRetreiver.apk?raw=true

image

### CODE REVIEW

**Comment by bombardier-gif** [comment]
I liked this idea. Let me take a deeper look at it in an hour.

File: README.md [hunk 1]
```diff
@@ -67,18 +72,43 @@ This is a fork over the neat https://github.com/pallupz/covid-vaccine-booking Th
 ![image](https://user-images.githubusercontent.com/83712877/117159863-4496a380-adde-11eb-8874-40cc6f851cf6.png)
 ![image](https://user-images.githubusercontent.com/83712877/117325821-b5a58c00-aeae-11eb-8156-2ea585a77834.png)
 
+##### Using the CoWIN OTP Retriever Android app
```
Comment (bombardier-gif) [inline README.md:75]: Can you please mention that you need to enable  APK installation through unknown sources. A link to a tutorial would suffice

**Comment by rohankelkar35** [comment]
Please update kvdb. Also would be nice to allow manual entry of kvdb incase of update

File: README.md (hunk 1 above)
Comment (ujaiswal) [reply README.md:75]: @bombardier-gif check now. I have added support for kvdb custom bucket key and updated the readme with turorial on how to enable apk installation through unknown sources

**Comment by bombardier-gif** [comment]
Great can you update the key to ASth4wnvVDPkg2bdjsiqMN instead? Then i will merge the PR

**Comment by ujaiswal** [comment]
@bombardier-gif I have updated the app default kvdb bucket key. Check now

**Comment by sumit2k21** [comment]
@ujaiswal Is it available for iPhones as well ? is there iOS app too ?

**Comment by ujaiswal** [comment]
@sumit2k21 I am not an iOS developer and I would not be able to implement an ios app similar to this one.

**Comment by bombardier-gif** [comment]
Hi @ujaiswal I am holding this PR for couple of hours. The reason is that the code became quite unstable in last 1 day due to PRs which were not well written and we are facing bunch of issues.

**Comment by bombardier-gif** [comment]
Thanks for the awesome work. I would suggest you keep an eye on the issues which might come in the next few days!
Cheers and stay safe :)

**Pull request merged**